.circleci/config.yml

```diff
@@ -185,16 +185,14 @@ jobs:
         description: 'Which config file to use'
         type: enum
         enum: ['prod', 'canary']
-    environment:
-      config: << parameters.config >>
     steps:
       - setup_vm
       - install_chrome
       - restore_karma_cache:
           cache_name: nomodule
       - run:
           name: 'Nomodule Tests (using << parameters.config >>-config.json)'
```

Comment:
nit: While we're here, can we shorten the parenthesized parts of these step names to just `(prod)` or `(canary)`? 2 main reasons:

- When the logs from this step are expanded on mobile, the header currently occupies 3 lines.
- The important info is prod vs. canary. The names of the config files are a detail of implementation.

Reply:
Done